node/rpcstack.go

```diff
@@ -197,6 +203,20 @@ func (h *httpServer) start() error {
 }
 
 func (h *httpServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Permit dumb empty requests for remote health-checks (AWS)
```

Comment:
Remove the AWS, in parenthesis here. We want to keep our logic cloud-agnostic in general, considering the OP Labs uses GCP.